package.json

```diff
@@ -58,5 +58,8 @@
         "extends": [
             "@commitlint/config-conventional"
         ]
+    },
+    "dependencies": {
```

Comment:
I think this should be a dev dependencies :thinking:

Reply:
I couldnt download git-cz without `--workspace-root` :/ thats why it added the dependencies, was there another command i could have done?